packages/components/components/elvis-modal/src/react/elvia-modal.tsx

```diff
@@ -66,16 +66,24 @@ export const Modal: FC<ModalProps> = function ({
   const hasSecondaryButton = !!secondaryButton || !!webcomponent?.getSlot('secondaryButton');
   const hasHeading = !!heading || !!webcomponent?.getSlot('heading');
 
+  const lang = useLanguage();
+
   const getAriaLabel = (): string => {
     if (heading && typeof heading === 'string') {
       return heading;
     } else if (webcomponent?.getSlot('heading')) {
-      return webcomponent.getSlot('heading').textContent ?? 'Forgrunnsvindu';
+      return (
+        webcomponent.getSlot('heading').textContent ?? (lang === 'no' ? 'Forgrunnsvindu' : 'Popup window')
+      );
     } else {
-      return 'Forgrunnsvindu';
+      return lang === 'no' ? 'Forgrunnsvindu' : 'Popup window';
     }
   };
 
+  const getCloseAriaLabel = (): string => {
+    return lang === 'no' ? 'Lukk modal' : 'Close modal';
+  };
+
```

Comment:
We should be consistent in naming the component when it is being read aloud.

- In Norwegian, we now call it both "Forgrunnsvindu" and "modal"
- In English, we now call it both "Popup window" and "modal"

I suggest we keep "Forgrunnsvindu" for Norwegian and "Modal" for English.

Reply:
I've changed the labels and it will now return Modal if the lang attribute is 'en' and Forgrunnsvindu if it is set to 'no'. Is this what you had in mind?
![modal_language_consistancy](https://github.com/user-attachments/assets/12f6a650-d6d6-46df-82bc-a7b3427cfe9b)

Reply:
Yes! Very nice